editor.planx.uk/src/@planx/components/PlanningConstraints/Public.tsx

```diff
@@ -284,19 +291,18 @@ export function PlanningConstraintsContent(
             >
               <ConstraintsList data={negativeConstraints} metadata={metadata} />
             </SimpleExpand>
-            <PlanningConditionsInfo />
+            <Disclaimer text={disclaimer} />
           </>
         )}
     </Card>
   );
 }
 
-const PlanningConditionsInfo = () => (
+const Disclaimer = (props: { text: string }) => (
   <WarningContainer>
     <ErrorOutline />
     <Typography variant="body1" ml={2} fontWeight={FONT_WEIGHT_SEMI_BOLD}>
-      This page does not include information about historic planning conditions
-      that may apply to this property.
+      {props.text || DEFAULT_PLANNING_CONDITIONS_DISCLAIMER}
```

Comment:
Since I've introduced `disclaimer` as a required prop, the `DEFAULT_` fallback is necessary here to ensure currently published services don't have a visual regression until the component is next updated